### PR TITLE
Update to Swift 3.1.1; add 7/5 snapshot to Travis; update copyright y…

### PR DESCRIPTION
**Pull request by quanvo87**
…ear; IBM-Swift/Kitura#1121

## Checklist:

- [ ] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA)
- [ ] If applicable, I have updated the documentation accordingly.
- [ ] If applicable, I have added tests to cover my changes.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,11 +17,14 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0-DEVELOPMENT-SNAPSHOT-2017-06-11-a
+      env: SWIFT_SNAPSHOT=4.0-DEVELOPMENT-SNAPSHOT-2017-07-05-a
```
Comment (youming-lin) [inline .travis.yml:20]: Why 07-05 snapshot? I thought we are standardizing on 06-29 for now?

File: Tests/LinuxMain.swift [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2016
+ * Copyright IBM Corporation 2017
```
Comment (youming-lin) [inline Tests/LinuxMain.swift]: Should be `2016, 2017`

File: .travis.yml (hunk 1 above)
Comment (quanvo87) [reply .travis.yml:20]: Is there an advantage to standardizing to one snapshot over just using the latest snapshot for whatever repo we are doing that day?

Comment (youming-lin) [reply .travis.yml:20]: Probably not a big deal, but just in case they introduce/revert a breaking change in between like the issue with tuples.

Comment (quanvo87) [reply .travis.yml:20]: Ok, moving forward I'll stick to 6/29

**Pull request merged**